packages/mui-material/src/SpeedDial/SpeedDial.test.js

```diff
@@ -424,6 +424,191 @@ describe('<SpeedDial />', () => {
     });
   });
 
+  describe('dial focus with slotProps.fab', () => {
```

Comment:
copied this entire this describe block and just replaced `FabProps` with `slotProps.fab` [here](https://github.com/mui/material-ui/pull/46508/files#diff-d919b96940de356147e91105461dd0bd710a0df6a9143aa00ba8b10a37ec499cR459)

https://github.com/mui/material-ui/blob/3fdcb916348633b8fa15e472391575c165ec2d40/packages/mui-material/src/SpeedDial/SpeedDial.test.js#L244